<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1590</width>
    <height>755</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>102</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>102</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>102</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>102</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Chekiang Sung</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <layout class="QGridLayout" name="Central_Grid" rowstretch="2,2,1,2,1,2,1,6,1,2" columnstretch="0">
      <item row="4" column="0">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0">
       <spacer name="HS1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="0">
       <layout class="QGridLayout" name="Grid_Slogan">
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="L_Slogan">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>標楷體</family>
            <pointsize>20</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>一 時 存 錢 一 時 爽 ， 一 直 存 錢 一 直 爽 ， 錢 多 就 比 誰 更 浪 ， 身 價 翻 倍 紅 不 讓</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="Grid_Time" columnstretch="1">
        <item row="0" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <spacer name="HS2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="Grid_Title">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="Title">
          <property name="font">
           <font>
            <family>SetoFont</family>
            <pointsize>60</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>樂來樂生氣銀行</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="Grid_Detail" columnstretch="1,1,2">
        <item row="0" column="1">
         <layout class="QGridLayout" name="Grid_Rate" rowstretch="0" columnstretch="0">
          <item row="0" column="0">
           <widget class="QLineEdit" name="LE_Account">
            <property name="font">
             <font>
              <pointsize>30</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>標楷體</family>
            <pointsize>40</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>交易帳戶</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QGridLayout" name="Grid_Group" columnstretch="1,1,0,2">
        <item row="0" column="2">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>標楷體</family>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string>萬元</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="L_Group">
          <property name="font">
           <font>
            <family>標楷體</family>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string>交易金額</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="LCD_Trade">
          <property name="font">
           <font>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="digitCount">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QGridLayout" name="Grid_Button">
        <item row="0" column="1">
         <widget class="QPushButton" name="BT_Back">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="text">
           <string>返回 RETURN</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTableWidget" name="Table">
          <property name="font">
           <font>
            <family>標楷體</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>交易時間</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>存入</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>領出</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>餘額</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,0,2">
        <item row="0" column="2">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>標楷體</family>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string>萬元</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>標楷體</family>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string>帳戶餘額</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="LCD_Money">
          <property name="font">
           <font>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="digitCount">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1590</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
